src-tauri/tauri.conf.json

```diff
@@ -88,7 +88,7 @@
         "withGlobalTauri": true,
         "windows": [
             {
-                "title": "Tari Universe (Alpha) | Testnet",
+                "title": "Tari Universe",
```

Comment:
This can stay. 

It's overridden in CI. It exists locally so anyone who builds a local build know it's not a real `release` build.